scripts/build/gen_relocate_app.py

```diff
@@ -243,6 +245,9 @@ def find_sections(filename: str) -> 'dict[SectionKind, list[OutputSection]]':
         out = defaultdict(list)
 
         for section in sections:
+            if len(symbol_filter) > 0 and not re.search(symbol_filter, section.name):
```

Comment:
nit: The first condition should not be needed, as long as `symbol_filter` is an actual string.
```suggestion
            if not re.search(symbol_filter, section.name):
```

Reply:
Fixed.